packages/services/api/src/modules/organization/resolvers.ts

```diff
@@ -255,6 +256,13 @@ export const resolvers: OrganizationModule.Resolvers = {
       };
     },
     async inviteToOrganizationByEmail(_, { input }, { injector }) {
+      await injector.get(InMemoryRateLimiter).check(
```

Comment:
Instead of raising HiveError we can also use the union type and return an error response

Reply:
I don't think we should, it's not a piece of logic that user in his/her daily workflow

Reply:
In that case - should we disclose that the request failed due to rate limit?

Reply:
just asking I am fine with merging either way

Reply:
User will see `Exceeded rate limit for inviting to organization by email.`

Reply:
I think it's not different from `x-rate-limit-retry-at` etc so yeah, we should show it